pyvista/plotting/renderers.py

```diff
@@ -417,6 +417,7 @@ def set_background(self, color, top=None, all_renderers=True):
         >>> plotter.set_background('black')
         >>> plotter.background_color
         Color(name='black', hex='#000000ff')
+        >>> plotter.close()
```

Comment:
This will work after https://github.com/pyvista/pyvista/pull/2907.

Reply:
Output on top of #2907:

![Uploading Screenshot 2022-06-29 at 19-55-09 set_background — PyVista 0.35.dev0 documentation.png…]()